test/projects/full/lib/uniform/blueprint.ex

```diff
@@ -24,9 +24,11 @@ defmodule Full.Uniform.Blueprint do
   base_files do
     cp "assets/static/images/#{app.extra[:logo_file]}.png"
     template "config/runtime.exs"
+    file "priv/wildcard_me/*.txt"
 
     if app.extra[:company] == :fake_co do
-      file [".dotfile", ".another-dotfile"]
+      file ".dotfile"
+      file ".another-dotfile"
```

Comment:
Note: The `[...]` syntax is no longer supported. If you do it, you get this output:

```
** (RuntimeError) `file` was not given a path or glob.

You provided:

    file [".dotfile", ".another-dotfile"]

Try something like this instead:

    file "path/to/a/file"

```